docs: add point about physical disks visualization in dashboard (#11122)
rook/rook: Documentation/Storage-Configuration/Monitoring/ceph-dashboard.md

  dashboard behind a proxy already served using SSL) by setting the `ssl` option
  to be false.

## Visualization of 'Physical Disks' section in the dashboard

Information about of physical disks is available only in [Rook host clusters](../../CRDs/Cluster/host-cluster.md).

> The Rook operator configuration have another couple of settings which are needed for showing this information:

Reviewer: ```suggestion
The Rook operator configuration has two settings which are needed for showing disk information:
```

> - `ROOK_ENABLE_DISCOVERY_DAEMON` must be set to `true`, to provide the dashboard the information about physical disks.

Reviewer: ```suggestion
- `ROOK_ENABLE_DISCOVERY_DAEMON`: Set to `true` to provide the dashboard the information about physical disks. The default is `false`.
```

> - `ROOK_DISCOVER_DEVICES_INTERVAL`is set by default to `60m`, this means that changes in the set of physical disks in the cluster will only be refreshed each 60 minutes.  

Reviewer: ```suggestion
- `ROOK_DISCOVER_DEVICES_INTERVAL`: The interval for changes to be refreshed in the set of physical disks in the cluster. The default is 60 minutes.
```

  dashboard behind a proxy already served using SSL) by setting the `ssl` option
  to be false.

## Visualization of 'Physical Disks' section in the dashboard

Information about physical disks is available only in [Rook host clusters](../../CRDs/Cluster/host-cluster.md).

> The Rook manager module is used by the dashboard to obtain the information about physical disks, but it is disabled by default, so displaying the dashboard 'Physical Disks' section will show an error message warning about the situation.

Reviewer: ```suggestion
The Rook manager module is required by the dashboard to obtain the information about physical disks, but it is disabled by default. Before it is enabled, the dashboard 'Physical Disks' section will show an error message.
```

To prepare the Rook manager module to be used in the dashboard, use the following commands in the Ceph toolbox:

```console

> $ ceph mgr module enable rook

Reviewer: Instead of the toolbox commands, it can actually be enabled with the cluster CR, and rook will take care of setting the backend.
```yaml
  mgr:
    modules:
      - name: rook
        enabled: true
```

$ ceph orch set backend rook
$ ceph orch status
  Backend: rook
  Available: Yes
```

> Once enabled and selected the Rook manager module as the orchestrator backend, it will be needed to adjust the Rook operator configuration. There are two settings which are needed for showing disk information:

Reviewer: ```suggestion
Once the Rook manager module is enabled as the orchestrator backend, there are two settings required for showing disk information:
```

  dashboard behind a proxy already served using SSL) by setting the `ssl` option
  to be false.

## Visualization of 'Physical Disks' section in the dashboard

Information about physical disks is available only in [Rook host clusters](../../CRDs/Cluster/host-cluster.md).

The Rook manager module is required by the dashboard to obtain the information about physical disks, but it is disabled by default. Before it is enabled, the dashboard 'Physical Disks' section will show an error message.

To prepare the Rook manager module to be used in the dashboard, modify your cluster CRD:

> ```console

Reviewer: ```suggestion
```yaml
```

  mgr:
    modules:
      - name: rook
        enabled: true

> ```

Reviewer: nit: add blank lines before and after the code snippets
